src/transportType.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum TransportTypeEnum {
```

Comment:
Using `Enum` as a suffix for an enum is not very common. Can simply be `TransportType` or `TransportKind` or something similar.